docs/UsingCypress.md

```diff
@@ -4,7 +4,7 @@ Our frontend testing is done with cypress.
 
 To run the tests locally:
 
-- start a server with `npm run dev -- --language c++ --noLocal` - this configuration ensures your setup is clean of any
+- start a server with `npm run dev -- --language c++ --no-local` - this configuration ensures your setup is clean of any
```

Comment:
Was this an actual change or just update of wrong documentation?

Reply:
An actual change